airbyte-cdk/java/airbyte-cdk/db-sources/src/main/kotlin/io/airbyte/cdk/integrations/source/relationaldb/state/SourceStateIterator.kt

```diff
@@ -71,6 +73,7 @@ open class SourceStateIterator<T>(
                 )
             }
             recordCount = 0L
+            LOGGER.info { "sending real state message: $finalStateMessageForStream" }
```

Comment:
nit: what does "real" mean here :-)

Reply:
debug log - will remove! thanks for catching it.